Templates/Files/Local Files/JSON to Zoho CRM - Customers/long-description.md

```diff
@@ -0,0 +1,65 @@
+This template reads a JSON file containing customer data from a local fileshare and either inserts or updates each customer in Zoho CRM as contacts.
+
+![Template](assets/JSON_to_Zoho_CRM_-_Customers.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Frends agent has access to the local fileshare containing the JSON file.
+- Zoho CRM client ID and client secret should be accessible and provided in the process variables to create the refresh token.
+- Zoho CRM refresh token has been generated and provided in the process variables. A refresh token can be generated with the corresponding template.
+Zoho CRM regional domains for account and API are adjusted accordingly.
```

Comment:
Sadly you still won't "get away" with this - you missed the `-` for the last bullet point! 😄

Also, I think the sentence is now a bit more weird: "[...] adjusted accordingly." - according to what? 😅 It isn't really clear in the sentence. Could you take one more look at that? 😄

Reply:
I can't believe how I missed that! It's been changed once more now 😄 And I also removed and rephrased the accordingly-phrase.